gravitee-exchange-controller/gravitee-exchange-controller-core/src/main/java/io/gravitee/exchange/controller/core/DefaultExchangeController.java

```diff
@@ -82,6 +90,13 @@ protected void doStart() throws Exception {
         super.doStart();
         controllerClusterManager.start();
         startBatchFeature();
+
+        primaryChannelEvictedTopic =
+            clusterManager.topic(identifyConfiguration.identifyName(PrimaryChannelManager.PRIMARY_CHANNEL_EVENTS_EVICTED_TOPIC));
+        primaryChannelEvictedSubscriptionId =
+            primaryChannelEvictedTopic.addMessageListener(message ->
```

Comment:
Didn't you want to call the listener on on the primary cluster node ?

Reply:
good point I forgot that 🙈